saskatoon/sitebase/templates/app/detail_views/organization/equipment_table.html

```diff
@@ -0,0 +1,43 @@
+{% load i18n %}
+{% load static %}
+<div class="search-engine-area mg-t-30">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+        <div class="data-table-list">
+          <div class="table-responsive">
+            <div>
+              <h4> {% trans Equipment %} </h4>
+              {% if perms.harvest.add_equipment and organization.is_equipment_point %}
+              <a href="{% url 'equipment-create' %}?pid={{ property.id }}">
+                <button title="New Equipment" class="btn btn-success notika-btn-success waves-effect">
+                  <i class="fa fa-plus"></i>
+                  {% trans "New Equipment" %}
+                </button>
```

Comment:
If you are to add a `New Equipment` button then I would ideally like to prefill and hide the `owner` field from the `EqupmentForm` so we make sure there is no confusion and the newly created equipment actually belongs to the organization. This can be dealt with in a separate PR if needed.

Reply:
For reference you can look at how the Property ID `pid` is passed to the `HarvestCreateView` when the `HarvestForm` is opened from within a property detail page:
https://github.com/LesFruitsDefendus/saskatoon-ng/blob/3308f376fa89b210b1be4ba02264f411967d0568/saskatoon/harvest/views.py#L115

Reply:
Another important point is that the `Property` field should be hidden from the form when it is called from an Organization details page.
Altogether I am wondering if a simpler inline form similar to the fruit distribution one would not be preferable here since there really aren't too many fields to fill:
- 1 equipment type autocomplete
- 1 description
- 1 count
https://github.com/LesFruitsDefendus/saskatoon-ng/blob/3308f376fa89b210b1be4ba02264f411967d0568/saskatoon/harvest/views.py#L275

Reply:
added the autofill and took out the `Property` field, but left the `Owner` field for UX. i think it's helpful for user to see what is being implied

i think the separate form is still a good idea for UX consistency. fruit inline form needed a refresh on submit too, so almost defeats the flow.

Reply:
Ok, I that case I suggest we do the following in the `EquipmentForm` : 
- change the label for "Owner" to "Equipment Point"
- change the `actor-autocomplete` for a `equipmentpoint-autocomplete`